modules/clientutils.js

```diff
@@ -242,14 +245,15 @@
          * @param  HTMLElement|String  form      A form element, or a CSS3 selector to a form element
          * @param  Object              vals      Field values
          * @param  Function            findType  Element finder type (css, names, xpath)
-         * @return Object                        An object containing setting result for each field, including file uploads
+         * @return Object                        An object containing setting result for each field,
+         *                                       including file uploads
          */
         this.fill = function fill(form, vals, findType) {
             /*eslint complexity:0*/
             var out = {
-                errors: [],
-                fields: [],
-                files:  []
+                "errors": [],
```

Comment:
No, please set
```"quote-props": ["as-needed", { "keywords": true }]``` in ```.eslintrc``` instead. This has the effect to reduce the change-set in this PR and probably the nice side-effect to get rid of lots of lint warnings in one go.